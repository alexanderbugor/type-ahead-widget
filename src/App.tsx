import React from 'react';

// eslint-disable-next-line react/prefer-stateless-function
class App extends React.Component {
  render() {
    return (
      <div>
        Here we go
      </div>
    );
  }
}

export default App;
