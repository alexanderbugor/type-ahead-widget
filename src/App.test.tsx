import React from 'react';
import { shallow } from 'enzyme';
import App from './App';

describe('App', () => {
  it('should render and match snapshot', () => {
    const root = shallow(<App />);

    expect(root).toMatchSnapshot();
  });
});
