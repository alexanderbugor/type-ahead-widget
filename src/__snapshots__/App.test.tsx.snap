// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render and match snapshot 1`] = `
<div>
  Here we go
</div>
`;
